config/prod.exs

```diff
@@ -9,3 +9,12 @@ config :logger, level: :info
 
 # Runtime production configuration, including reading
 # of environment variables, is done on config/runtime.exs.
+config :sentry,
+  dsn: System.get_env("SENTRY_DSN"),
```

Comment:
@danielnv18 `config/prod.exs` is executed at compile time, but ENV vars are only available at runtime. This key will need to move to `runtime.exs`. We also want to require it's presence, so should use `System.fetch_env!()` instead